dot/parachain/collator-protocol/validator_side.go

```diff
@@ -474,3 +558,15 @@ func (cpvs CollatorProtocolValidatorSide) processMessage(msg interface{}) error
 
 	return nil
 }
+
+func newFetchedCollationInfo(candidateReceipt parachaintypes.CandidateReceipt) fetchedCollationInfo {
+	candidateHash, _ := candidateReceipt.Hash()
```

Comment:
Might not wanna ignore the error here.

Reply:
Thanks! Addressed this in latest commit!